dev-requirements.txt

```diff
@@ -7,7 +7,7 @@ mypy==0.790
 sphinx
 sphinx-rtd-theme~=0.4
 sphinx-autodoc-typehints
-pytest!=5.2.3
+# pytest!=5.2.3
```

Comment:
Why is this changed?

Reply:
Ah I get it, somehow I slept through all of the drama happened over atomicwrites package.

Reply:
do you mean this? https://github.com/untitaker/python-atomicwrites/issues/61

Reply:
Please revert this since the packages are uploaded.